autoswitch_virtualenv.plugin.zsh

```diff
@@ -142,6 +143,7 @@ function _default_venv()
         _maybeworkon "$(_virtual_env_dir "$AUTOSWITCH_DEFAULTENV")" "virtualenv"
     elif [[ -n "$VIRTUAL_ENV" ]]; then
         deactivate
+        unset venv_dir venv_name venv_path venv_type
```

Comment:
I'm not entirely convinced this is needed. But if we decide to keep it, a comment explaining the reason why these are being unset within the code would be good to have.

Reply:
I have removed this line by making them local instead.